qc_grader/challenges/quantum_explorers23/badge_6.py

```diff
@@ -167,9 +170,19 @@ def grade_badge6_ex15(answer15: List[int]) -> None:
 ##### coding question
 
 @typechecked
-def grade_badge6_code(ans: List) -> None:
+def grade_badge6_code(tsp: Tsp, qubo: QuadraticProgram, result: SamplingVQEResult) -> None:
+
+    x = tsp.sample_most_likely(result.eigenstate)
+    z = tsp.interpret(x)
+    adj_matrix = nx.to_numpy_array(tsp.graph)
+
+    answer = {
+        'is_feasible': qubo.is_feasible(x),
+        'solution objective': tsp.tsp_value(z, adj_matrix)
```

Comment:
Doesn't matter too much but for consistency could add an underscore `solution_objective`

Reply:
Thank you! Fixed in https://github.com/qiskit-community/Quantum-Challenge-Grader/pull/119/commits/293e3db1f9d65c690da687cd2d28e781efdebd6e